system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -308,8 +309,25 @@ pub type TrustedTeleporters = (
 	IsForeignConcreteAsset<FromSiblingParachain<parachain_info::Pallet<Runtime>>>,
 );
 
-/// We allow locations to alias into their own child locations.
-pub type Aliasers = AliasChildLocation;
+/// Defines all global consensus locations that the asset hub is allowed to alias into.
+pub struct PolkadotOrEthereumGlobalConsensus;
+impl Contains<Location> for PolkadotOrEthereumGlobalConsensus {
+	fn contains(location: &Location) -> bool {
+		matches!(location.unpack(), (2, [GlobalConsensus(network_id)]) | (2, [GlobalConsensus(network_id), ..])
+			if matches!(*network_id, NetworkId::Polkadot | NetworkId::Ethereum { chain_id: 1 }))
```

Comment:
to_polkadot::EthereumNetwork

@brenzi very nit, we have constants:
```
			if matches!(*network_id, to_polkadot::PolkadotNetwork::get() | to_polkadot::EthereumNetwork::get()))
```

Reply:
I see. But already merged.... Maybe fix the next time we touch this?